src/components/service.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+    font-family: 'Inter';
+    src: local('Inter'), url(../fonts/Inter/Inter-VariableFont_slnt,wght.ttf) format('truetype');
+    font-weight: 100 1000;
+	font-stretch: 25% 151%
+  }
+
+div{
+    font-family: Inter;
+    font-weight: 350;
+    text-align: left;
+}
+
+.heading{
+    margin: auto;
+    width: 50%;
+    padding: 10px;
+    text-align: center;
+    top: 100px;
+    color: rgb(255, 255, 255);
+}
+
+.textBackground {
+    padding: 20px;
+    position: relative;
+    max-width: 1000px;
+    width: 80%;
+    background: rgb(33, 36, 41);
+    box-shadow: rgb(0 0 0 / 1%) 0px 0px 1px, rgb(0 0 0 / 4%) 0px 4px 8px, rgb(0 0 0 / 4%) 0px 16px 24px, rgb(0 0 0 / 1%) 0px 24px 32px;
+    border-radius: 30px;
+    top: 3vh;
+    height: 27vh;
+    margin: auto;
+    color: rgb(195, 197, 203);
+}
+
+.serviceBackground{
+    background-color: rgb(44, 47, 54);
+    min-height: 100vh;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: radial-gradient(50% 50% at 50% 50%, rgba(33, 114, 229, 0.1) 0%, rgba(33, 36, 41, 0) 100%);
+    background-size: 'cover';
+
+}
+
+.tryButton {
```

Comment:
There's some things here which I don't think are necessary
```
position: relative
bottom: 0px
```
I don't see any reason why this needs to have a relative position
```
-webkit-box-pack
-webkit-box-align
```
Looking at the docs https://developer.mozilla.org/en-US/docs/Web/CSS/box-pack
There is a big warning "This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behaviour may change in the future."
Apparently this has been replaced by flexbox, which you're already using anyway